protocol/contracts/beanstalk/field/FieldFacet.sol

```diff
@@ -191,10 +198,14 @@ contract FieldFacet is Invariable, ReentrancyGuard {
         // are harvestable.
         harvestablePods = s.sys.fields[fieldId].harvestable.sub(index);
```

Comment:
Is there a test that verifies if you harvest a slashed/burned plot, then anyone can harvest it and also this function returns 0? I am hopefully misunderstanding, but it seems like currently this function might not return 0 for slashed plots and beans might get sent out for them

Reply:
harvesting a slashed/burned plot does not return 0 here, but should not send out beans. i feel this is tested in TransmitToFork.t.sol, but not sure that it specifically checks the bean balance to confirm nothing is being sent.